airflow/utils/log/file_task_handler.py

```diff
@@ -140,32 +141,6 @@ def _interleave_logs(*logs):
         last = v
 
 
-def _ensure_ti(ti: TaskInstanceKey | TaskInstance | TaskInstancePydantic, session) -> TaskInstance:
```

Comment:
i am not sure why move this to task context logger. maybe you could share your reasoning?  file task handler seems lower level if anything.  but yea if you are going to remove the single underscore, please add `:meta private:` so this does not become public interface unnecessarily.

Reply:
i would personally rather have duplication, or "internal" importing of single underscore methods, than unnecessarily expanding public interface.  it's a real pain when we have to worry about making updates to things that no user should be using.

Reply:
Sure, I can put it back where it was